Sources/PackageLoading/TargetSourcesBuilder.swift

```diff
@@ -274,8 +290,9 @@ public struct TargetSourcesBuilder {
 
             // If a resource is both inside a localization directory and has an explicit localization, it's ambiguous.
             guard implicitLocalization == nil || explicitLocalization == nil else {
-                let relativePath = path.relative(to: targetPath)
-                self.observabilityScope.emit(.localizationAmbiguity(path: relativePath, targetName: target.name))
+                // FIXME: Restore error
```

Comment:
need to be addressed?

Reply:
Yep, was just trying to get something roughly working to see if this approach makes sense to everyone.

Reply:
I think it does.  This is along the lines of what I was thinking, though I hadn't looked into it as closely yet as you have.  Eventually we probably want to rework TargetSourcesBuilder to carry through some of the manifest information, as you said in one of the comments, but I think there is value in some support here even if all cases aren't yet handled.